modules/cloud-config-container/simple-nva/main.tf

```diff
@@ -19,35 +19,80 @@ locals {
     files                = local.files
```

Comment:
if these are not used directly in the tf code but just from other locals they should be prefixed with an underscore

Reply:
I'd also update the year of the copyright at the beginning of the file

Reply:
Updated copyright, regarding underscore everything is aligned with the same logic of all the other modules, updating this should probably be part of a new PR. What do you think?

Reply:
I got what you mean and aligned on the last commit, maybe we should align as well all the others module in the next days (I added this task to my backlog) :)